core/trino-main/src/main/java/io/trino/operator/MultiChannelGroupByHash.java

```diff
@@ -980,55 +990,54 @@ public GroupByIdBlock getResult()
         }
     }
 
-    private abstract class LowCardinalityDictionaryWork<T>
-            implements Work<T>
+    /**
+     * Returns an array containing a single Page position occurrence that corresponds to the
```

Comment:
Simplify
`a single Page position occurrence that corresponds` -> `a position that corresponds`